components/suggest/src/store.rs

```diff
@@ -405,7 +405,7 @@ where
                     let data = self.settings_client.get_attachment(&attachment.location)?;
                     writer.write(|dao| {
                         dao.put_icon(icon_id, &data)?;
-                        dao.handle_ingested_record(record)
+                        dao.handle_ingested_record(record, false)
```

Comment:
I wonder if we could make this an enum instead of a Boolean, or split up `handle_ingested_record` into `handle_ingested_record` and `handle_ingested_record_with_unknown_fields`?

`handle_ingested_record(record, false)` feels a bit too close to a ["Boolean trap"](https://ariya.io/2011/08/hall-of-api-shame-boolean-trap) to me—it's not super obvious from context what `false` means.